doc/classes/Array.xml

```diff
@@ -192,6 +192,21 @@
 				Removes the first occurrence of a value from the array.
 			</description>
 		</method>
+		<method name="filter">
+			<return type="Array">
+			</return>
+			<argument index="0" name="method" type="Callable">
+			</argument>
+			<description>
+				Calls the provided [Callable] for each element in array and removes all elements for which the method returned [code]false[/code].
+				[codeblock]
+				func _ready():
+				    print([1, 2, 3].filter(Callable(self, "remove_1"))) # prints [2, 3]
+
+				func remove_1(number):
+				    return number != 1
```

Comment:
You need to close all the codeblocks:
```suggestion
				    return number != 1
				[/codeblock]
```

Reply:
You only closed for the `reduce` doc. Need to close it here and in `map` as well. That's what's causing the static checks to fail.